frontend/src/pages/destructionlist/review/DestructionListReview.action.ts

```diff
@@ -123,3 +135,31 @@ export async function destructionListRejectListAction({
   }
   return redirect("/");
 }
+
+/**
+ * React Router action, user intends to complete a co-review.
+ * @param request
+ * @param params
+ */
+export async function destructionListCompleteCoReviewAction({
+  request,
+}: ActionFunctionArgs) {
+  const { payload } = await request.json();
+  const { comment, destructionList } =
+    payload as ReviewDestructionListCompleteCoReviewPayload;
+
+  const data: CoReview = {
+    destructionList: destructionList,
+    listFeedback: comment,
+  };
+
+  try {
+    await createCoReview(data);
+  } catch (e: unknown) {
+    if (e instanceof Response) {
+      return await (e as Response).json();
+    }
+    throw e;
+  }
```

Comment:
should this also have the "handle error" thing so that a modal shows that something went wrong?

Reply:
Yes, which is what `submitAction` does: https://github.com/maykinmedia/open-archiefbeheer/blob/main/frontend/src/hooks/useSubmitAction.ts#L68